guides/orders/payment-states.html.markdown

```diff
@@ -0,0 +1,37 @@
+# Payment states
+
+<!-- TODO:
+  This article is a stub. It may need to be merged with another orders article
+  in the future.
+-->
+
+You can check the state of the `Spree::Payment` associated with any order:
```

Comment:
There are two ways that the state of payments are represented. There is the state of the payment status for the order, which is what is described below and the state of an individual payment.

As an example:

An order for $100 has a single payment of $50 that is complete. The status of the one payment is complete, but the order is in a payment state of balance_due.

If the order has multiple payments:

- $80 - failed
- $80 - completed
- $80 - completed

The order will be in a payment_state of credit owed.

Refunds play in to this as well as those can bring the payment state back in to a state of pair if the customer overpaid in the above case. This is complicated enough that it needs an explanation of what the difference between order `payment_state` and payment's `state` is.

Reply:
I am going to include an example like this, thank you.